patterns/template-archive-portfolio.php

```diff
@@ -8,7 +8,7 @@
  */
 ?>
 
-<!-- wp:group {"style":{"position":{"type":"sticky","top":"0px"}},"layout":{"type":"default"}} -->
+<!-- wp:group {"layout":{"type":"default"}} -->
 <div class="wp-block-group">
```

Comment:
Is the group still needed? I thought it was only added for the sticky option.

Reply:
I think we need it for the layout, but we can test

Reply:
I just tested, it can be removed and still works fine. I'm working on a commit now.